Code/6-wire-version/src/main.cpp

```diff
@@ -21,78 +21,113 @@ void setup() {
 			// int dsp_audio_pin 	= D6 
 			// );
 	//example: bwc.begin(D1, D2, D3, D4, D5, D6, D7);
-  updateMqttTimer.attach(600, sendMQTTsetFlag);     //update mqtt every 10 minutes. Mqtt will also be updated on every state change
-  updateWSTimer.attach(2.0, sendWSsetFlag);         //update webpage every 2 secs plus state changes
-  periodicTimer.attach(60, periodicTimerCallback);  //things to do once in a while
-  LittleFS.begin(); // needs to be loaded here for reading the wifi.json
+
+  // check things every 60 seconds
+  periodicTimer.attach(60, []{ periodicTimerFlag = true; });
```

Comment:
@visualapproach this is the timer to check the wifi to be connected?

Reply:
Yes.